### PR TITLE
search results: render Mathjax in the results list and grid

### PR DESCRIPTION
**Pull request by anikachurilova**
* closes https://github.com/CERNDocumentServer/cds-rdm/issues/198

### CODE REVIEW

**Review by zzacharo — APPROVED** [review]

File: src/lib/components/ResultsMultiLayout/ResultsMultiLayout.js [hunk 1]
```diff
@@ -38,15 +38,19 @@ ResultsMultiLayout.defaultProps = {
...
+  }, []);
+
   return (
     <Overridable
       id={buildUID("ResultsMultiLayout.element", overridableId)}
       layout={layout}
     >
       {layout === "list" ? (
-        <ResultsList overridableId={overridableId} />
+        <ResultsList overridableId={overridableId} mathjaxCallback={handleMathjaxCallback}/>
```
Comment (kpsherva) [inline src/lib/components/ResultsMultiLayout/ResultsMultiLayout.js:51]: isn't it a bit too specific? Shouldn't we pass a callback (as a name) and use it for mathjax?

Comment (zzacharo) [reply src/lib/components/ResultsMultiLayout/ResultsMultiLayout.js:51]: I agree with @kpsherva , I missed that :) Probably we should rename it to something `onRender`

File: src/lib/components/ResultsMultiLayout/ResultsMultiLayout.js [hunk 2]
```diff
@@ -7,17 +7,17 @@
...
 import { ResultsGrid } from "../ResultsGrid";
 import { ResultsList } from "../ResultsList";
 import { ShouldRender } from "../ShouldRender";
 
-function ResultsMultiLayout({ loading, totalResults, currentLayout, overridableId }) {
+function ResultsMultiLayout({ loading, totalResults, currentLayout, overridableId, callback }) {
   return (
     <ShouldRender condition={currentLayout != null && !loading && totalResults > 0}>
-      <Element layout={currentLayout} overridableId={overridableId} />
+      <Element layout={currentLayout} overridableId={overridableId} callback={callback} />
```
Comment (zzacharo) [inline src/lib/components/ResultsMultiLayout/ResultsMultiLayout.js:20]:
```suggestion
      <Element layout={currentLayout} overridableId={overridableId} onRender={callback} />
```

File: src/lib/components/ResultsGrid/ResultsGrid.js [hunk 3]
```diff
@@ -32,11 +45,13 @@ ResultsGrid.propTypes = {
   loading: PropTypes.bool.isRequired,
   totalResults: PropTypes.number.isRequired,
   results: PropTypes.array.isRequired,
+  onRender: PropTypes.func,
 };
 
 ResultsGrid.defaultProps = {
   resultsPerRow: 3,
   overridableId: "",
+  onRender: undefined,
```
Comment (zzacharo) [inline src/lib/components/ResultsGrid/ResultsGrid.js:54]: documentation: could you maybe check if we have already documentation for this component on our [RSK docs](https://github.com/inveniosoftware/react-searchkit/tree/master/docs) and document also the new property?

**Review by zzacharo — APPROVED** [review]

**Review by zzacharo — APPROVED** [review]

File: src/lib/components/ResultsMultiLayout/ResultsMultiLayout.js [hunk 4]
```diff
@@ -33,20 +34,20 @@ ResultsMultiLayout.propTypes = {
 ResultsMultiLayout.defaultProps = {
   currentLayout: null,
   overridableId: "",
+  onResultsRendered: () => {},
 };
 
-const Element = ({ layout, overridableId }) => {
+const Element = ({ layout, overridableId, onResultsRendered }) => {
```
Comment (ntarocco) [inline src/lib/components/ResultsMultiLayout/ResultsMultiLayout.js:40]: Sorry, last thing: the Overridable cmp *always* require that you pass *all* params. `onResultsRendered` needs to be passed `onResultsRendered={onResultsRendered}`.
Double-check the other cmps above too.

**Review by ntarocco — APPROVED** [review]

**Pull request merged**